utilities/ft_version.m

```diff
@@ -141,9 +141,12 @@
   
 else
   % get it from the Contents.m file in the FieldTrip release
-  a = ver(ftpath);
-  ftver = a.Version;
-  
+  if ~isdeployed
```

Comment:
This difference seems a bit out of place for this PR, probably it needs to be removed.